Makefile

```diff
@@ -93,6 +93,10 @@ site-run-server:
 site-check:
 	./scripts/check-site.sh
 
+site-gen-sidebar:
```

Comment:
We shouldn't keep adding top-level make commands. What's the set of steps a developer needs to do locally to render the site?

There should be a single top-level command for the site.

Reply:
Keeping the site generation target separate for use with different methods of building the site, but added it to the site running target so a developer only needs to run `make site-run-server`.